test/select.test.js

```diff
@@ -49,6 +49,255 @@ describe('record selection', function() {
+            res.json({
+                records: [
+                    {
+                        id: 'recordA',
+                        fields: {Name: 'Rebecca'},
+                        createdTime: '2020-04-20T16:20:00.000Z',
+                    },
+                ],
+                offset: 'offsetABC',
+            });
+        });
+
+        return expect(
+            airtable
+                .base('app123')
+                .table('Table')
+                .select()
+                .all('monkeys')
+        ).rejects.toThrow(/must be a function/);
+    });
+
+    it('selects records and paginates the selection without a done function', function(done) {
+        var iterationCounter = 0;
+
+        testExpressApp.set('handler override', function(req, res) {
+            expect(req.method).toBe('GET');
+            expect(req.url).toBe('/v0/app123/Table?');
+            res.json({
+                records: [
+                    {
+                        id: 'recordA',
+                        fields: {Name: 'Rebecca'},
+                        createdTime: '2020-04-20T16:20:00.000Z',
+                    },
+                ],
+                offset: 'offsetABC',
+            });
+        });
+
+        return airtable
+            .base('app123')
+            .table('Table')
+            .select()
+            .eachPage(
+                function page(records, fetchNextPage) {
+                    if (iterationCounter === 0) {
+                        expect(records.length).toBe(1);
+                        records.forEach(function(record) {
+                            expect(record.id).toBe('recordA');
+                            expect(record.get('Name')).toBe('Rebecca');
+                        });
+
+                        testExpressApp.set('handler override', function(req, res) {
+                            expect(req.method).toBe('GET');
+                            expect(req.url).toBe('/v0/app123/Table?offset=offsetABC');
+                            res.json({
+                                records: [
+                                    {
+                                        id: 'recordB',
+                                        fields: {Name: 'Clinton'},
+                                        createdTime: '2020-04-20T16:20:00.000Z',
+                                    },
+                                ],
+                            });
+                        });
+                    } else if (iterationCounter === 1) {
+                        expect(records.length).toBe(1);
+                        records.forEach(function(record) {
+                            expect(record.id).toBe('recordB');
+                            expect(record.get('Name')).toBe('Clinton');
+                        });
+                    }
+                    iterationCounter++;
+                    fetchNextPage();
+                }).then(function() {
+                    expect(iterationCounter).toBe(2);
```

Comment:
Sorry for the noisy CI report; we probably don't need to lint across versions of Node.js. [This patch](https://github.com/Airtable/airtable.js/pull/190) ought to help.

Reply:
Thanks!